Packs/StellarCyber/pack_metadata.json

```
@@ -0,0 +1,40 @@
+{
+    "name": "Stellar Cyber",
+    "description": "Integration to retrieve and update cases from the Stellar Cyber platform.",
+    "support": "partner",
+    "currentVersion": "1.0.0",
+    "author": "Stellar Cyber",
+    "url": "https://success.stellarcyber.ai/",
+    "email": "support@stellarcyber.ai",
+    "categories": [
+        "Analytics & SIEM"
+    ],
+    "certification": "certified",
+    "tags": [
+        "Alerts",
+        "Incident Handling",
+        "Security",
+        "Security Analytics"
+    ],
+    "useCases": [
+        "Case Management"
+    ],
+    "keywords": [],
+    "marketplaces": [
+        "xsoar",
+        "marketplacev2"
```

To fix the following errors:
```
Packs/StellarCyber/Layouts/layoutscontainer-StellarCyber_Case.json: [LO107] - The following invalid types were found in the layout: ['linkedIncidents', 'evidence', 'team', 'evidenceBoard', 'relatedIncidents', 'canvas'].
Those types are not supported in XSIAM, remove them or change the layout to be XSOAR only.

Packs/StellarCyber/Layouts/layoutscontainer-StellarCyber_Case.json: [LO107] - The following invalid types were found in the layout: ['linkedIncidents', 'evidence', 'team', 'evidenceBoard', 'relatedIncidents', 'canvas']. Those types are not supported in XSIAM, remove them or change the layout to be XSOAR only
```
You can either change the marketplace as I suggested here or remove the invalid types so it would be acceptable in Xsiam. 

 

```suggestion
    "marketplaces": [
        "xsoar",
```